RELEASE.md

```diff
@@ -6,7 +6,16 @@ The Kubernetes Windows GMSA project is released on an as-needed basis. The proce
 1. All [OWNERS](OWNERS) must LGTM this release issue
 1. An OWNER runs `git tag -s $VERSION` from `master` branch and pushes the tag with `git push $VERSION`
 1. An OWNER promotes the `gcr.io/k8s-staging-gmsa-webhook/k8s-gmsa-webhook` image built the tagged commit.
-    1. Insuctions TBD
+    1. Follow setup steps for `kpromo` from [here](https://github.com/kubernetes-sigs/promo-tools/blob/main/docs/promotion-pull-requests.md#preparing-environment) if needed
+    1. Tag the desired container image in the [staging registry](https://console.cloud.google.com/gcr/images/k8s-staging-gmsa-webhook/GLOBAL) as `$VERSION`
```

Comment:
not sure I follow this. do we manually tag this image in gcr or are we grabing the tag from gcr we want and set it to `$VERSION`?

Reply:
Yes, manually tag.